### PR TITLE
Spelling and grammar corrections - M through Z

### PR DESCRIPTION
**Pull request by bchavez**
Continuation PR of #13683, divided up as reasonably byte size digestible chunks. :)

:briefcase: :necktie: ***["Taking care of business every day... Taking care of business every way..."](https://www.youtube.com/watch?v=mmwic9kFx2c)***

### CODE REVIEW

File: src/mscorlib/src/System/Diagnostics/Eventing/EventSource_CoreCLR.cs [hunk 1]
```diff
@@ -77,7 +77,7 @@ public static void SetCurrentThreadActivityId(Guid activityId)
         /// This API marks the current thread as working on activity 'activityID'. It returns 
         /// whatever activity the thread was previously marked with. There is a convention that
         /// callers can assume that callees restore this activity mark before the callee returns. 
-        /// To encourage this this API returns the old activity, so that it can be restored later.
+        /// To encourage this API returns the old activity, so that it can be restored later.
```
Comment (danmosemsft) [inline src/mscorlib/src/System/Diagnostics/Eventing/EventSource_CoreCLR.cs]: I think this was correct before albeit missing comma

File: src/mscorlib/src/System/Diagnostics/Eventing/EventSource_CoreCLR.cs [hunk 2]
```diff
@@ -77,7 +77,7 @@ public static void SetCurrentThreadActivityId(Guid activityId)
         /// This API marks the current thread as working on activity 'activityID'. It returns 
         /// whatever activity the thread was previously marked with. There is a convention that
         /// callers can assume that callees restore this activity mark before the callee returns. 
-        /// To encourage this this API returns the old activity, so that it can be restored later.
+        /// To encourage this, this API returns the old activity, so that it can be restored later.
```
Comment (bchavez) [inline src/mscorlib/src/System/Diagnostics/Eventing/EventSource_CoreCLR.cs]: @danmosemsft fantastic find! Thanks! PR updated.

**Comment by danmosemsft** [comment]
Thanks @bchavez . BTW, when you make a fix to a PR, if you do a regular commit rather than a squash or force push then it's easy for the reviewer to see only what changed. 😸

**Comment by bchavez** [comment]
Ah, thanks for the tip @danmosemsft ! I'll avoid force-pushing the PRs next time. :sunglasses:

**Pull request merged**